code/modules/religion/sects/shadow_sect.dm

```diff
@@ -42,8 +45,10 @@
 
 /datum/religion_sect/shadow_sect/on_select(atom/religious_tool, mob/living/user)
 	. = ..()
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
```

Comment:
you don't need to take it from GLOB. this src itself is already what it takes.

Reply:
So,` GLOB.blahblah` to just `= src?`

Reply:
Removing anyway  as we dont need the sect.murder var, hopefully